pkg/controller/controlplane/valuesprovider.go

```diff
@@ -192,10 +190,32 @@ func getControlPlaneChartValues(
 }
 
 // getControlPlaneShootChartValues collects and returns the control plane shoot chart values.
-func getControlPlaneShootChartValues(
+func (vp *valuesProvider) getControlPlaneShootChartValues(
+	cp *extensionsv1alpha1.ControlPlane,
 	cluster *extensionscontroller.Cluster,
 	credentials *packet.Credentials,
 ) (map[string]interface{}, error) {
 
-	return map[string]interface{}{}, nil
+	_, err := vp.decodeControlPlaneConfig(cp)
+	if err != nil {
+		return nil, errors.Wrapf(err, "decoding control plane config")
+	}
```

Comment:
Why was this added? It seems to not have any effect?

Reply:
It was related to storage. When I removed the storage, we didn't need the return value, but I forgot to delete the whole thing.

Removed, push on its way.